tests/tests.py

```
@@ -90,19 +92,19 @@ def test_hstore_serializer(self):
         self.assertIs(s['boolean_true'], False)
         self.assertEqual(s['char'], 'char')
         self.assertEqual(s['text'], 'text')
-        self.assertEqual(s['date'], d.date)
-        self.assertEqual(s['datetime'], d.datetime)
-        self.assertEqual(s['decimal'], 2.0)
+        self.assertEqual(s['date'], str(d.date))
+        self.assertEqual(s['datetime'], str(d.datetime.isoformat()))
+        self.assertEqual(s['decimal'], '2.00')
         self.assertEqual(s['email'], 'test@test.com')
         self.assertEqual(s['ip'], '10.10.10.10')
         self.assertEqual(s['url'], 'http://test.com')
         # should be hidden
-        self.assertTrue('data' not in s)
-    
+        # self.assertTrue('data' not in s) # Why?
```

Ok I see, you could also just uncomment the line and let the test fail. We can take care of this in the subsequent commits.